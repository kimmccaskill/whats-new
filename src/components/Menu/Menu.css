.menu {
  color: #fff;
  width: 20%;
  display: flex;
  flex-direction: column;
  align-items: center;
}

.title {
  padding: 10%;
}

.menu-btns {
  width: 90%;
  height: 50px;
  margin-bottom: 5%;
  font-size: 1.3em;
  background: #141527;
  border: none;
  color: #fff;
  cursor: pointer;
}
