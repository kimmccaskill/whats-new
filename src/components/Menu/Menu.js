import React from 'react';
import './Menu.css'

const Menu = (props) => {
  return(
    <aside className='menu'>
      <h1 className="title">What's New?</h1>
        <button key={Math.random()} className="menu-btns" type="button" value='Entertainment' onClick={props.changeCategory}>Entertainment</button>
        <button key={Math.random()} className="menu-btns" type="button" value='Health' onClick={props.changeCategory}>Health</button>
        <button key={Math.random()} className="menu-btns" type="button" value='Local' onClick={props.changeCategory}>Local</button>
        <button key={Math.random()} className="menu-btns" type="button" value='Science' onClick={props.changeCategory}>Science</button>
        <button key={Math.random()} className="menu-btns" type="button" value='Technology' onClick={props.changeCategory}>Technology</button>
    </aside>
  )
}

export default Menu;
