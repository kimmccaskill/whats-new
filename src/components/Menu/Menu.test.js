import React from 'react';
import { shallow } from 'enzyme';
import Menu from './Menu';
import local from '../../data/local';

describe('Menu', () => {
  it('should match the snapshot with all data passed in correctly', () => {
    const wrapper = shallow(<Menu
      changeCategory={jest.fn()}
      data={local}
    />);

    expect(wrapper).toMatchSnapshot();
  });
});
