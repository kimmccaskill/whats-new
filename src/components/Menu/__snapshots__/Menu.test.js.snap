// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menu should match the snapshot with all data passed in correctly 1`] = `
<aside
  className="nav-bar"
>
  <h1>
    What's New?
  </h1>
</aside>
`;
